solr/webapp/web/js/angular/controllers/analysis.js

```diff
@@ -137,16 +168,24 @@ solrAdminApp.controller('AnalysisController',
         }
 
         if (fieldOrType == "fieldname") {
-          $location.search("analysis.fieldname", name);
-          $location.search("analysis.fieldtype", null);
+            $location.search("analysis.fieldname", name);
```

Comment:
I don't love the whitespace change, however I see we have a werid mix of two and four space indentations on if/else throughout this method!   We need some code styling tools for jS ;-)

Reply:
yep, fixed the formatting. tried as much as possible to keep the original style